core/templates/dev/head/pages/exploration-player-page/templates/exploration-player-suggestion-modal.directive.html

```diff
@@ -5,6 +5,7 @@ <h3>Suggest a Change</h3>
 <div class="modal-body protractor-test-exploration-suggestion-modal">
   <!-- ng-model needs to bind to a property of an object on the scope (the property cannot sit directly on the scope). -->
   <ck-editor-4-rte ng-show="showEditor" ng-model="suggestionData.suggestionHtml"></ck-editor-4-rte>
+  <ck-editor-5-rte ng-show="showEditor" ng-model="suggestionData.suggestionHtml"></ck-editor-5-rte>
```

Comment:
Don't you need ng-ifs here and elsewhere?

Reply:
same here